pkg/controller/migcluster/predicate.go

```diff
@@ -14,6 +15,9 @@ type ClusterPredicate struct {
 }
 
 func (r ClusterPredicate) Create(e event.CreateEvent) bool {
+	if !common.IsInSingletonNamespace(e.Meta.GetNamespace()) {
```

Comment:
Can you help explain to me what exactly this is doing? I'm not super familiar with the predicate code.